backend/globaleaks/handlers/base.py

```diff
@@ -145,7 +146,7 @@ def get_session(self):
 
         if session.user_role != 'whistleblower' and \
            self.state.tenants[1].cache.get('log_accesses_of_internal_users', False):
-             self.request.log_ip_and_ua = True
+            self.request.log_ip_and_ua = True
```

Comment:
@joeman65 @alessiofranceschini: This seems to be just stilistic  code review, please provide this separately to not misk code review of our code with the features you need to implement.